class.jetpack-gutenberg.php

```diff
@@ -334,6 +334,14 @@ public static function enqueue_block_editor_assets() {
 			plugins_url( $blocks_dir . '/', JETPACK__PLUGIN_FILE )
 		);
 
+		$is_subscriptions_active = Jetpack::is_module_active( 'subscriptions' );
+		if ( $is_subscriptions_active ) {
+			// Get number of subscribers
+			$subscriptions = new Jetpack_Subscriptions_Widget();
+			$subscriber_info = $subscriptions->fetch_subscriber_count();
```

Comment:
Does it make sense to convert fetch_subscriber_count to a static method so that we can just call it directly?

Reply:
good call, but the constructor does some magic, like adding xmlrpc endpoints that might be needed 🤔 

but this make me realize that we should call `Jetpack_Subscriptions_Widget::init()` instead of creating a new instance